parl/common/logging.py

```diff
@@ -1,19 +1,20 @@
+import glog
```

Comment:
Maybe we should change "prints" in all the files to glog?

Reply:
done